cli/flags.rs

```diff
@@ -696,7 +709,16 @@ instead of being alone on the next line (does not apply to self closing elements
 }
 
 fn repl_subcommand<'a, 'b>() -> App<'a, 'b> {
-  SubCommand::with_name("repl").about("Read Eval Print Loop")
+  SubCommand::with_name("repl")
+    .about("Read Eval Print Loop")
+    .arg(
+      Arg::with_name("v8-flags")
+        .long("v8-flags")
+        .takes_value(true)
+        .use_delimiter(true)
+        .require_equals(true)
+        .help("Set V8 command line options. For help: --v8-flags=--help"),
```

Comment:
Can you create a function v8_flags_arg() (similar to reload_arg()) so that this bit of code can be shared between repl and run